pkg/controller/direct/bigqueryreservation/assignment_controller.go

```diff
@@ -18,7 +18,8 @@ import (
 	"context"
 	"fmt"
 
-	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/bigqueryreservation/v1alpha1"
+	krmv1alpha1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/bigqueryreservation/v1alpha1"
+	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/bigqueryreservation/v1beta1"
```

Comment:
I suspect it would be easier to keep `krm` as the "primary" version and use longer aliases for the "secondary" version - fewer code changes

Reply:
Well, no. Using `krmv1beta1` will end up with more changes. Because eventuall (once all resources are in Beta), we want to change `krmv1beta1` to `krm`, so that's about twice about of code changes.

Reply:
I think when we promote BigQueryReservationAssignment to v1beta1, _that_ is when krm is repointed to v1beta1